ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -255,23 +277,33 @@ export default class TestDataModal extends React.Component {
       this.addRelatedApplications(params);
     }
 
-    if (filterText) {
-      await this.updateFilterText(filterText);
-    }
+    this.applyFilters(filterText);
```

Comment:
You don't really need to provide the `filterText` argument.
The function is capable of getting it from the state. It already does that with the `tag` filter.

Reply:
I think it is going to add some extra work for `applyFilters` if we do it this way because the function is used by `InputFilter` as well. It would have to decide if the `filterText` is from `InputFilter` or can be taken from `state`.